vispy/visuals/volume.py

```diff
@@ -753,7 +745,11 @@ def method(self, method):
         if 'u_attenuation' in self.shared_program:
             self.shared_program['u_attenuation'] = None
 
-        self.shared_program.frag = frag_dict[method]
+        # TODO: $sample needs to be unset and re-set, since it's present inside the snippets.
```

Comment:
This isn't actually a TODO right? More of a note?

Reply:
yeah that's correct, code works as is

Reply:
```suggestion
        # $sample needs to be unset and re-set, since it's present inside the snippets.
```